templates/base/challenge_card.html.jinja

```diff
@@ -0,0 +1,116 @@
+                {% if challenge.user_has_submitted %}
+                <!-- User has submitted - show green checkmark -->
+                <span class="text-byte-green-200 font-medium flex items-center gap-1">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" fill="none" viewBox="0 0 24 24"
+                        stroke="currentColor" stroke-width="2" class="inline-block">
+                        <path d="M20 6L9 17l-5-5" stroke="currentColor" stroke-width="2" stroke-linecap="round"
+                            stroke-linejoin="round" />
+                    </svg>
+                    Submitted
+                </span>
+                {% else %}
+                <!-- User hasn't submitted - show time info -->
+                {% if challenge.is_post_mortem %}
+                <span class="text-byte-brown-200 flex items-center gap-1">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" fill="none" viewBox="0 0 24 24"
+                        stroke="currentColor" stroke-width="2" class="inline-block">
+                        <circle cx="12" cy="12" r="10" stroke="currentColor" stroke-width="2" fill="none" />
+                        <path d="M12 6v6l4 2" stroke="currentColor" stroke-width="2" stroke-linecap="round"
+                            stroke-linejoin="round" />
+                    </svg>
+                    Ended
+                </span>
+                {% elif challenge.post_mortem_date %}
+                <span class="text-amber-500 flex items-center gap-1">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" fill="none" viewBox="0 0 24 24"
+                        stroke="currentColor" stroke-width="2" class="inline-block">
+                        <circle cx="12" cy="12" r="10" stroke="currentColor" stroke-width="2" fill="none" />
+                        <path d="M12 6v6l4 2" stroke="currentColor" stroke-width="2" stroke-linecap="round"
+                            stroke-linejoin="round" />
+                    </svg>
+                    {{ challenge.post_mortem_date | format_date }}
+                </span>
+                {% else %}
+                <span class="text-byte-brown-200 flex items-center gap-1">
+                    N/A
+                </span>
+                {% endif %}
+
+                {% endif %}
+
+                <!-- Submission count -->
+                <span class="text-byte-brown-200">
+                    {% if challenge.submission_count > 0 %}
+                    {{ challenge.submission_count }} submission{% if challenge.submission_count != 1 %}s{% endif %}
+                    {% else %}
+                    No submissions yet
+                    {% endif %}
+                </span>
+                <!-- Score (hidden temporarily, for filtering) -->
+                {% if challenge.score %}
+                <span class="hidden" data-challenge-score>{{ challenge.score }}</span>
```

Comment:
You can do `data-challenge-score="{{challenge.score}}" instead. This can also be done on the div for the entire challenge so you don't need an entirely separate span